### ISSUE TITLE
The EnableVisualStyle setting apparently does not work.

### ISSUE THREAD

**Issue by Anixx** [otya128/winevdm#1293]
Tried with Aldus Page Maker, wanted to run it with visual style.

**Comment by otya128** [comment]
If you are using the classic style, then you cannot enable the visual style.

**Comment by Anixx** [comment]
No, I run it when all programms are themed. Still, Winevdm remains in classic theme.
